tests/dashboards/api_tests.py

```diff
@@ -109,21 +112,35 @@ def create_dashboards(self):
         with self.create_app().app_context():
             dashboards = []
             admin = self.get_user("admin")
-            for cx in range(DASHBOARDS_FIXTURE_COUNT - 1):
-                dashboards.append(
-                    self.insert_dashboard(f"title{cx}", f"slug{cx}", [admin.id])
+            charts = []
+            half_dash_count = round(DASHBOARDS_FIXTURE_COUNT / 2)
+            for cx in range(DASHBOARDS_FIXTURE_COUNT):
+                dashboard = self.insert_dashboard(
+                    f"title{cx}",
+                    f"slug{cx}",
+                    [admin.id],
+                    slices=charts if cx < half_dash_count else [],
                 )
+                if cx < half_dash_count:
+                    chart = self.insert_chart(f"slice{cx}", [admin.id], 1, params="{}")
```

Comment:
Non blocking. Having one or a bunch of dashboards with more then 1 chart could make a better GET test